rustls/src/msgs/deframer/outer.rs

```diff
@@ -0,0 +1,236 @@
+use core::mem;
+
+use crate::error::{Error, InvalidMessage};
+use crate::msgs::codec::Reader;
+use crate::msgs::message::{
+    read_opaque_message_header, InboundOpaqueMessage, MessageError, HEADER_SIZE,
+};
+
+/// A deframer of outer TLS messages.
```

Comment:
In the process of removing error fusing from here, the `OuterDeframer` disappeared. So I moved the `DeframerIter` back down into `deframer/mod.rs`. The remaining docstrings talk about "TLS wire messages" rather than "outer".